src/app/context/CurrencyContext.tsx

```diff
@@ -0,0 +1,138 @@
+import { useState, useEffect, createContext, useContext } from "react";
```

Comment:
Would be good to have Jest tests for these functions. I could support with that.
Or can we re-use tests from https://github.com/AryanJ-NYC/bitcoin-conversion ?

Reply:
The tests would be different from the ones in the lib as it'd be testing within the CurrencyContext, do you have any examples for writing tests against a context? I'm not sure how to structure those.

Reply:
Does this help?  
https://testing-library.com/docs/example-react-context/

Reply:
Is there a reason we really need to use context? Could we replace this with just using utilities? (without relying on React functionality)

Reply:
Good point, this will also make testing easier.